src/data-display/tables/data-table/PDataTable.vue

```diff
@@ -169,11 +169,13 @@ interface TableField extends DataTableFieldType {
 
 const getChildFields = (field: DataTableFieldType): DataTableFieldType[]|undefined => field.children?.map(child => ({ sortable: true, ...child }));
 
+const isFieldSortable = (sortable: boolean | undefined): boolean => [undefined, true].includes(sortable);
```

Comment:
How about changing `isFieldSortable` to check both sortable prop and field's sortable?

```suggestion
const isFieldSortable = (sortable: boolean | undefined): boolean => {
  if (props.sortable) {
    if (!soratable) return false;
    return true;
  }
  return false;
}
```

Reply:
isFieldSortable function is located outside of setup function. The isFieldSortable function and getTableField function need arguments for props.sortable. would it be okay?

Reply:
yes! it would be okay!